Style.css

```diff
@@ -50,30 +50,29 @@ h1 {
   padding-top: 20%;
 }
 
-.connect {  
-  font-weight: 500;  
-  font-size: 16px;  
-  line-height: 24px;  
-  font-style: normal;  
-  color: #7F8CFF;  
-  text-transform: uppercase; 
-  margin-bottom: 0; 
+.connect {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  font-style: normal;
+  color: #7f8cff;
+  text-transform: uppercase;
+  margin-bottom: 0;
   text-decoration: none;
-  }
+}
 
-.socials {  
-  display: flex;  
-  justify-content: space-between;  
-  max-width: 150px; 
-  padding-inline-start: 0; 
+.socials {
+  display: flex;
+  justify-content: space-between;
+  max-width: 150px;
+  padding-inline-start: 0;
   padding-bottom: 15px;
-  }
+}
 
-  li {
-    list-style: none;
-  }
+li {
+  list-style: none;
+}
 
-  .union {
-   color: #6070FF;
-  }
-  
\ No newline at end of file
+.union {
+  color: #6070ff;
+}
```

Comment:
- [ Optional  ] As stated the project requirement You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly adjust  the margins (top, right) for the hamburger button defined in the design.

The Figma design the toolbar (or header):

![Capture](https://user-images.githubusercontent.com/93598189/206129991-435bca48-6dd7-4fed-a3d7-df97b604ce23.PNG)

Your the toolbar (or header) Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/206130272-63b3e218-2b7e-43a7-966d-18cd15aac2e1.PNG)